pdm lock update

muhammad-ali-e opened the pull request:
## What

- Updated PDM lock to reflect new unstract core changes

## Why

- Due to missing pakage `kombu` error 

## How

- updated pdm lock with command `pdm update`

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)

- No. It contains a package addition to pdm lock to reflect new changes from log publisher package

## Database Migrations

- 

## Env Config

- 

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions

-

## Notes on Testing

-

## Screenshots

## Checklist

I have read and understood the [Contribution Guidelines]().

ritwik-g reviewed (commented):
@muhammad-ali-e `unstract-core` is being used in 
- backend
- worker
- workflow-execution
- prompt-service
Can you confirm if this is update in all these services?

muhammad-ali-e commented:
@ritwik-g Yes we already updated in backend and worker along with the parent PR. 
Workflow execution is also using backend.

muhammad-ali-e commented:
> @ritwik-g Yes we already updated in backend and worker along with the parent PR. Workflow execution is also using backend.


@ritwik-g Yes we already updated in backend and worker along with the parent PR.
Workflow execution is also using backend.

muhammad-ali-e commented:
> @muhammad-ali-e `unstract-core` is being used in
> 
> * backend
> * worker
> * workflow-execution
> * prompt-service
>   Can you confirm if this is update in all these services?


@ritwik-g Yes we already updated in backend and worker along with the parent PR.
Workflow execution is also using backend.
